src/components/dialog-manager-test.js

```diff
@@ -129,6 +129,16 @@ describes.realWin('DialogManager', {}, (env) => {
     expect(graypaneStubs.destroy).to.not.be.called;
   });
 
+  it('should catch view error', async () => {
+    const view = {
+      whenComplete: () =>
+        Promise.reject(new DOMException('cancel', 'AbortError')),
+    };
+    expect(dialogManager.handleCancellations(view)).to.eventually.throw(
```

Comment:
```suggestion
    await expect(dialogManager.handleCancellations(view)).to.eventually.throw(
```

This needs an `await` keyword, otherwise the result is ignored

Reply:
Doesn't the eventually catch that?

Reply:
Adding in await gives me a strange test error, it fails with `cancel`.

Reply:
Added an await with your suggestion offline!